### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by dharmit**
This README is mostly copied over from the original README in master
branch. Few changes have been made to match the current service based on
OpenShift.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,184 +1,139 @@
...
-```bash
-$ yum install -y docker git centos-release-openshift-origin
-$ yum install -y origin-clients
-```
+- Scan the image for updates, fixes, capabilities and push it to a public
+  registry (by default, http://registry.centos.org)
+- Automatically rebuild when a change is detected within the repository. Such
+  as an update in base image (`FROM` in Dockerfile) or a `git push` to the
+  project's git repository
+- Notifications / alerts regarding scan results (by e-mail)
```
Comment (navidshaikh) [inline README.md]: regarding builds status and scan results

File: README.md [hunk 2]
```diff
@@ -1,184 +1,139 @@
...
+    well as branch for any changes. This triggers a new build on **OpenShift**
+    when a change is pushed.
+    
+    Changes to the application's repository, update to the base image or any
+    RPMs that are part of the image will trigger a new build
+
+3. **Building the image**
+
+    The container image is built within OpenShift and then pushed to an
+    internal registry. This results in an image tagged `image:test`.
```
Comment (navidshaikh) [inline README.md]: remove
"This results in an image tagged `image:test`"

File: README.md [hunk 3]
```diff
@@ -1,184 +1,139 @@
...
+
+4. **Scan and analyze the image**
+
+    Scanning happens by running scripts in the container image to check for:
+    - `yum` updates
+    - updates for packages installed via `pip`, `npm` and `gem`
+    - capabilities of the container created from resulting container image by
+      analyzing `RUN` label in the Dockerfile
+    - verify the installed RPMs by running `rpm -Va` in the resulting container
```
Comment (navidshaikh) [inline README.md]: *Verify the RPM installed libraries in binaries in the resulting container.

Remove the `rpm -Va` command

File: README.md [hunk 4]
```diff
@@ -1,184 +1,139 @@
...
+      image
+
+5. **Push to the public registry (https://registry.centos.org)**
+
+    Finally, the image is re-tagged to its final name based on the value within
```
Comment (navidshaikh) [inline README.md]: No - we don't re-tag the iamge now. We can re-word it.

"After successful scans, the container image is pushed to the registry."

**Comment by bamachrn** [comment]
#dotests

**Comment by bamachrn** [comment]
LGTM, merging

**Pull request merged**